ORM/ORMFundamentals/Repositories/OrderRepository.cs

```diff
@@ -0,0 +1,57 @@
+﻿using Microsoft.EntityFrameworkCore;
+using ORMFundamentals.Data;
+using ORMFundamentals.Entities;
+using System.Linq.Expressions;
+
+namespace ORMFundamentals.Repositories;
+
+public class OrderRepository : IOrderRepository
+{
+    private readonly AppDbContext _context;
+
+    public OrderRepository(AppDbContext context)
+    {
+        _context = context;
+    }
+
+    public async Task<IEnumerable<Order>> GetAllOrders()
+    {
+        return await _context.Orders.ToListAsync();
```

Comment:
Just a small advice: `async/await` keywords add some extra code logic during compilation into CIL with adding additional variables and state options to control method execution. It is called "machine state". In general, if you don't need to await for result complete to continue work with its content or you just put it into return, then I would strongly recommend to avoid adding `async/await`. Caller side will await if it is needed.
So, your final implementation should look like:
```suggestion
    public Task<IEnumerable<Order>> GetAllOrders()
    {
        return _context.Orders.ToListAsync();
```